src/core/Form/Textarea/__snapshots__/Textarea.test.tsx.snap

```diff
@@ -129,6 +129,7 @@ exports[`snapshot default structure should match snapshot 1`] = `
   -ms-flex-direction: column;
   flex-direction: column;
   color: hsl(0,0%,16%);
+  width: 290px;
```

Comment:
Have you tested that it's still possible to override this with css? It should be, but just to make sure.

Reply:
It works with the following example:

```css
.fi-textarea.ta--custom.ta--custom { 
  width: 500px;
}
```
And giving the `ta--custom` className for Textarea:
```jsx
className="ta--custom"
```